src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -143,6 +148,22 @@ public static void setStandaloneConfigIgnoredToRepoConfigs(
         configs.stream().forEach(config -> config.setStandaloneConfigIgnored(isStandaloneConfigIgnored));
     }
 
+    /**
+     * Ensures that the {@code displayName} of each {@code RepoConfiguration} is unique.
+     */
+    public static void makeDisplayNamesUnique(List<RepoConfiguration> configs) {
+        Map<String, Integer> uniqueNames = new HashMap<>();
+        for (RepoConfiguration config : configs) {
+            if (!uniqueNames.containsKey(config.getDisplayName())) {
+                uniqueNames.put(config.getDisplayName(), 1);
+            } else {
+                String currName = config.getDisplayName();
+                config.appendUniqueIdToDisplayName("(" + uniqueNames.get(currName).toString() + ")");
```

Comment:
I think the parentheses should be handled in `appendUniqueIdToDisplayName` instead